openid-connect-client/src/main/java/org/mitre/openid/connect/client/service/impl/StaticClientConfigurationService.java

```diff
@@ -66,7 +65,7 @@ public RegisteredClient getClientConfiguration(ServerConfiguration issuer) {
 		return clients.get(issuer.getIssuer());
 	}
 
-	@PostConstruct
+	@Override
```

Comment:
Why was this change made? We moved to using PostConstruct many years ago, so I'm not sure why we'd move back.